azure-functions/acbs-function/api.js

```diff
@@ -24,23 +25,48 @@ const getACBS = async (apiRef) => {
       headers: {
         'Content-Type': 'application/json',
       },
-    }).catch((err) => ({
-      status: !!err.response
-        ? err.response.status
-        : err,
-    }));
-    return response;
+    }).catch((e) => {
```

Comment:
should this be a `try-catch` block instead? I think we have a mix of `try-catch` and `axios.catch` throughout the code

Reply:
This will will suffice, as it will catch any error thrown.